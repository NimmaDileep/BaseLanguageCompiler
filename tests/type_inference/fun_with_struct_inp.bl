struct Dogs {
    age: Int,
    name: String,
    isHappy: Boolean
}

fun dog(){
    return Dogs(1, "Jackie", true);
}

print(dog());


