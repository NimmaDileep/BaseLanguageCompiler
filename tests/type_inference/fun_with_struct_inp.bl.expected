Dogs{age=1,name=Jackie,isHappy=true}
